### DISCUSSION TITLE
Bandoiler needs an improvement

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#15271] [Ideas and suggestions]
Currently I think they are underused, one potential change I seen tossed around is to have a big weapon slot for them and decrease the amount of items they can hold to compensate (ie. 1 (big) weapon slot, 3 ammo slots). 

This will make them more useful in the case synergizing with large weapons like HMG and giving them an edge over the more generalist toolbelt where you can hold other small items instead of just ammo.

https://discord.com/channels/103209684680323072/1276853412516073533

**Comment by Zimon3283** [comment]
Agreed
